### PR TITLE
Added sounds in game(with minor designs changes)

### PR DESCRIPTION
**Pull request by a-dzemiankova**

### CODE REVIEW

File: alien_invasion.py [hunk 1]
```diff
@@ -27,6 +30,8 @@ def __init__(self):
         self._create_fleet()
 
         self.play_button = Button(self, "Play")
+        self.message = Message(self, "Game over:(", self.play_button)
```
Comment (Vovass) [inline alien_invasion.py:33]: variable must be named game_over

File: sound_effects.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        self.shoot = "core_files/sounds/shoot.ogg"
+        self.collision = "core_files/sounds/collision.ogg"
+        self.fleet = "core_files/sounds/new_fleet.ogg"
+        self.ship_hit = "core_files/sounds/ship_hit.ogg"
+        self.game_over = "core_files/sounds/game_over.ogg"
+
+    def play_bg_music(self):
+        pygame.mixer.music.load(self.bg_music)
+        pygame.mixer.music.set_volume(0.4)
+        pygame.mixer.music.play(-1)
```
Comment (Vovass) [inline sound_effects.py:18]: static params must be determinate in init block

File: sound_effects.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        self.bg_music = "core_files/sounds/background.mp3"
+        self.shoot = "core_files/sounds/shoot.ogg"
+        self.collision = "core_files/sounds/collision.ogg"
+        self.fleet = "core_files/sounds/new_fleet.ogg"
+        self.ship_hit = "core_files/sounds/ship_hit.ogg"
+        self.game_over = "core_files/sounds/game_over.ogg"
+
+    def play_bg_music(self):
+        pygame.mixer.music.load(self.bg_music)
+        pygame.mixer.music.set_volume(0.4)
```
Comment (Vovass) [inline sound_effects.py:17]: static params must be determinate in init block

File: alien_invasion.py (hunk 1 above)
Comment (a-dzemiankova) [reply alien_invasion.py:33]: done

**Pull request merged**